src/common/buffer.cc

```diff
@@ -1350,7 +1350,7 @@ static simple_spinlock_t buffer_debug_lock = SIMPLE_SPINLOCK_INITIALIZER;
 
   // -- buffer::list --
 
-  buffer::list::list(list&& other)
+  buffer::list::list(list&& other) noexcept
     : _buffers(std::move(other._buffers)),
```

Comment:
@tchaikov 
I think std::list move constructor is not marked noexcept, so I was wondering if it's safe to be marking buffer::list::list(list&& other) noexcept ?